megamek/src/megamek/client/bot/PhysicalCalculator.java

```diff
@@ -281,19 +282,30 @@ public static PhysicalOption getBestPhysical(Entity entity, IGame game) {
         }
 
         for (Entity target : game.getEntitiesVector()) {
-
+            // don't consider myself
             if (target.equals(entity)) {
                 continue;
             }
+            
+            // don't consider friendly targets
             if (!target.isEnemyOf(entity)) {
                 continue;
             }
+            
+            // don't consider targets not on the board
             if (target.getPosition() == null) {
                 continue;
             }
+            
+            // don't consider targets beyond melee range
             if (Compute.effectiveDistance(game, entity, target) > 1) {
                 continue;
             }
+            
+            // don't bother stomping mechwarriors
+            if (target instanceof MechWarrior) {
```

Comment:
What about abandoned crews? Does this also handle that?

Reply:
Probably, MechWarrior inherits from EjectedCrew.

Reply:
Wait... it would need to be EjectedCrew if we also wanted the bot to ignore them, instead of just MechWarrior, given MechWarrior extends EjectedCrew.

So, this should be swapped to target instanceof EjectedCrew if we don't want it to attack general ejected crews.

Reply:
Good to know for future reference